src/newrelic/collector/event_data.go

```diff
@@ -223,6 +224,13 @@ func (daemonConfig *EventHarvestConfig) UnmarshalJSON(b []byte) error {
 	rawLimits := rawConfig.HarvestLimits
 	var harvestConfig EventConfigs
 
+	// MSF REMOVE POSSIBLY FOR RELEASE
+	if rawLimits.LogEventData != nil {
+		log.Debugf("unmarshal response collector suggested eventharvestconfig rawLimits (logging) = " + strconv.Itoa(*rawLimits.LogEventData))
+	} else {
+		log.Debugf("unmarshal response collector did not supply suggested eventharvestconfig rawLimits (logging)!")
+	}
+
```

Comment:
I would suggest removing these lines if there isn't a change made to the program flow itself (conditionally setting variables, etc) to avoid potentially cluttering up debug logs, depending on how often this function is called.

```suggestion
```

Reply:
I removed the comment but would like to keep this debug message in for troubleshooting.  It only occurs once when the daemon connects so the impact on performance and log file size should be quite small.